constructors/aarch64/construct.yaml

```diff
@@ -0,0 +1,20 @@
+name: conda_bootstrap
+version: 1.0.0
+
+channels:
+  - https://conda.anaconda.org/c4aarch64
+  - https://conda.anaconda.org/jjhelmus/label/aarch64_bootstrap
+
+conda_default_channels:
+  - https://conda.anaconda.org/c4aarch64
+  - https://conda.anaconda.org/jjhelmus/label/aarch64_bootstrap
+
+specs:
+  - python 3.7
```

Comment:
I think this assumes it is 3.7.0.

THat is fine on @jjhelmus's channels, but conda-forge now builds 3.7.1, so (shameless self plug here) https://anaconda.org/archiarm/ contains that.